libbeat/template/template.go

```diff
@@ -41,6 +45,39 @@ func New(beatVersion string, beatName string, esVersion string, config TemplateC
 		pattern = name + "-*"
 	}
 
+	compiler := func(key string, ops []fmtstr.VariableOp) (fmtstr.FormatEvaler, error) {
```

Comment:
For now to keep it as simple as possible I suggest to return an error in case any ops are used.